recipes/default.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+include_recipe 'runit' if node['logstash']['init_style'] == 'runit' || !SimpleLogstashCookbook::LogstashServiceSystemd.provides?(node, 'logstash_service')
```

Comment:
This is magic. I prefer to consider this cookbook like "library" cookbook. So you can deal with that in your wrapper cookbook. You even don't have to call `logstash_service` but implement your own.